model_compiler/python/nnir_to_nnef.py

```diff
@@ -127,7 +127,7 @@ def main():
     inputFolder = sys.argv[1]
     outputFolder = sys.argv[2]
     print('reading NNIR model from ' + inputFolder + ' ...')
-    graph = IrGraph()
+    graph = IrGraph(True)
```

Comment:
Codacy found an issue: ['IrGraph' may be undefined, or defined from star imports: nnir (F999)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5754497)